### ISSUE TITLE
Allow Ctrl+Alt+<keystoke>

### ISSUE THREAD

**Issue by brknsoul** [stefansundin/duplicate-tab#21]
Currently the extension only allows Ctrl, Ctrl+Shift, Alt, or Alt+Shift, but not Ctrl+Alt.

**Comment by stefansundin** [comment]
I don't think this is something I can control, unfortunately. Firefox has an API to set the shortcut, so perhaps there is something that can be done using that.

In version v1.5.1 and older, the Firefox version of this extension actually let you configure the shortcut using that API, in the extension options. However, I removed that in v1.5.2 since it is less work for me to just tell the user to go to the central place where the user can configure all extension shortcuts. So you could try to download v1.5.1 and load that manually in `about:debugging`. https://github.com/stefansundin/duplicate-tab/archive/refs/tags/v1.5.1.zip

If you are using Chrome then I think you are out of luck. And the idea above probably won't work.
